templates/web/base/admin/report_edit.html

```diff
@@ -41,17 +47,55 @@
 <li>[% loc('Body:') %]
   [% IF problem.bodies_str %]
     [% FOREACH body IN problem.bodies.values %]
+      [% SET body_printed = 1 %]
       <a href="[% c.uri_for('body', body.id) %]">[% body.name | html %]</a>
       [%- ',' IF NOT loop.last %]
     [% END %]
+    [% IF NOT body_printed %]
+      <em>[% problem.bodies_str %]</em>
+    [% END %]
   [% ELSE %]
     <em>[% loc('None' ) %]</em>
   [% END %]
-  ([% loc('other areas:') %] [% problem.areas | remove('^,') | remove( ',$' ) %])</li>
+  ([% loc('other areas:') %] [% problem.areas | remove('^,') | remove( ',$' ) | replace(',', ', ') %])</li>
```

Comment:
This looks nicer wrapped in a `<span class='sm'>`, it takes up a lot of space otherwise.